megatron/model/flash_attention.py

```diff
@@ -5,7 +5,7 @@
 import torch.nn.functional as F
 
 from flash_attn import flash_attn_triton
-import flash_attn_cuda
+import flash_attn_2_cuda as flash_attn_cuda # For flash_attn version 2.1.1
```

Comment:
Is this backwards-compatible with old flash_attn installations or will this break environments?

Reply:
Ok it's not backwards-compatible. We'll just drop support for old flash attn and announce it.